### ISSUE TITLE
Ability to mark a message as unread

### ISSUE THREAD

**Issue by danrabbit** [elementary/mail#258]

**Comment by davidmhewitt** [comment]
This is possible via the flag button in the headerbar, is this more of a design question of where else to expose the option?

**Comment by danrabbit** [comment]
Ah yeah there's no description ha. I'm guessing this was probably about marking read/unread using the icon in the conversation list

**Comment by davidmhewitt** [comment]
But given that the icon is invisible when the message is read, pretty hard to know what to click on to mark it as unread again?

**Comment by danrabbit** [comment]
Yeah, I believe in the old mail we showed `mail-read-symbolic` on row hover, but I don't have it handy to compare at the moment

**Comment by danrabbit** [comment]
This is probably not that high of a priority since we can mark read/unread from the mark menu

**Comment by Y0hy0h** [comment]
Just want to chime in that I didn't expect the flag to contain the "mark as unread" option. The other two options there are immediate, they don't have a submenu. So maybe that's why the flag didn't feel as though it contained this sort of option.

I'm pretty sure that as you described, you could previously click on the empty but outlined unread indicator. That is a reasonably discoverable method in my opinion.
